### DISCUSSION TITLE
Error with LDT testcases

### DISCUSSION THREAD

**Discussion by MariRiba** [NASA-LIS/LISF#1430] [Q&A] [answered]
Hi 
I have installed LDT and LIS but when I am running testcases, I am getting the below error:

[ldtlog.zip](https://github.com/NASA-LIS/LISF/files/12910837/ldtlog.zip)

#  - - - - - - - - - -  Albedo Parameters - - - - - - - - - - - - -
 Param_Check: Average max snow albedo values to be calculated ...
 Reading ./INPUT/LS_PARAMETERS/noah_2dparms/maximum_snow_albedo.hdf
 [ERR] MAX_SNOW_ALBEDO characteristics do not match expections!
  Rank is            2 , expected 2
  First dimension is    538705920 , expected         7200
  Second dimension is    269352960 , expected         3600
  Data type is            5 , expected            5  or         4101
 [ERR] Program stopping ...
 endrun is being called

What might be the issue here?
Is there a problem with the executable or with the " maximum_snow_albedo.hdf " file ?

Note: LIS executable is running properly without error.

**Comment by dmocko** [comment, marked as answer]
Hi @MariRiba 

As you can see in discussion #1405 - HDF-4 is required to read the maximum_snow_albedo dataset.

If you really can't compile HDF-4, you can edit the ldt.config file and use `"CONSTANT"` for your `Max snow albedo data source:`.

However, you will not get identical output between your run and the testcase target_output.
